Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -86,8 +86,12 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<Cursor<Site>, SearchViewController
         }
 
         // If the pre-path component (including the scheme) starts with the query, just use it as is.
-        let prePathURL = (url as NSString).substring(with: match.rangeAt(0))
+        var prePathURL = (url as NSString).substring(with: match.rangeAt(0))
         if prePathURL.startsWith(query) {
+            // Trailing slashes in the autocompleteTextField cause issues with Swype keyboard. Bug 1194714
+            if prePathURL.endsWith("/") {
+                prePathURL.removeLast()
```

Comment:
When I try to compile this, I get `Value of type 'String' has no member 'removeLast'`